### PR TITLE
docs: update `README` of the plugin

### PR DESCRIPTION
**Pull request by BryanttV**
### Description
This PR updates the documentation of the plugin.

### CODE REVIEW

**Review by Ian2012 — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -1,88 +1,236 @@
...
-|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge| |status-badge|
 
 Purpose
 *******
 
-Open edX plugin that includes a custom teams API
+Open edX plugin that includes a custom teams API, which extends the
+functionality of the Open edX default teams API. This plugin includes endpoints
+to create, delete, and list topics, and to add users (members) to teams.
```
Comment (mariajgrimaldi) [inline README.rst]: Can we add in what capacity is the team's API extended?

Comment (BryanttV) [reply README.rst]: Done

File: README.rst [hunk 2]
```diff
@@ -1,88 +1,246 @@
...
 Purpose
 *******
 
-Open edX plugin that includes a custom teams API
+Open edX plugin that includes a custom teams API, which extends the
+functionality of the Open edX default teams API. Compared to the default teams
+API, this plugin includes new endpoints to create and delete a topic and extend
+the functionality of list topics endpoint to include the list of teams of each
+topic. Also, extends the functionality of add a user to a team endpoint to add
+multiple users to a team using a list of usernames.
```
Comment (mariajgrimaldi) [inline README.rst]:
```suggestion
Open edX plugin that includes a custom teams API, which extends the
functionality of the Open edX teams API. Compared to the existing teams
API, this plugin includes new views to create and delete a topic and extends
listing topics to include the list of teams within each
topic. It also extends the functionality of adding a user to a team so
multiple users can be added to a team by using a list of usernames.
```

**Review by mariajgrimaldi — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (BryanttV) [reply README.rst]: Done

**Pull request merged**